spec.template.yaml

```diff
@@ -68,6 +99,23 @@ paths:
       description: Calls the MW API siteinfo action and returns the response
       produces:
         - application/json
+      parameters:
```

Comment:
I think that describing parameters like this was only added in 3.0 spec version, in 2.0 it's not supported.

Reply:
No?  https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#operationObject says that operation objects can have  a 'parameters' field that can take reference objects, and the 2.0 version of the online editor (http://editor2.swagger.io/) seems fine with it.

Reply:
Oh ok. Didn't know that. We've been describing parameters inline mostly, good to know.

I assume you've checked how the swagger-ui looks with these changes and all good? If so LGTML

Reply:
Yes, the swagger-ui still looks good.